.changeset/breezy-cats-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Export RawConfig and RawEnvironment types from Wrangler
```

Comment:
```suggestion
feat: Export RawConfig and RawEnvironment types from Wrangler
```

(if I remember correctly the wrangler team had the convention to follow conventional commit message formats here)

Reply:
We did, but we dropped it a few months ago

Reply:
😮 ah ok, good to know 🙂